modules/installation-initializing.adoc

```diff
@@ -62,6 +62,7 @@ it to install multiple clusters.
 +
 [IMPORTANT]
 ====
-The `install-config.yaml` file is consumed during the installation process. If
-you want to reuse the file, you must back it up now.
+The `install-config.yaml` file is consumed during the installation process.
+If you want to reuse the file, back it up now. If you lose the file, you can recover it
```

Comment:
Honestly, I wouldn't add this information. This isn't the point where a user would actually need to know this command, and we've told them twice to back it up. I'd defer the bug until we add a troubleshooting section. (@vikram-redhat, FYI)

Reply:
I concur.

Reply:
The note is already there, so I guess there is nothing to do for this bug?

Reply:
Not at this time. If you're collecting issues for a troubleshooting section, this would be a lovely addition to that list.

Reply:
@vikram-redhat What's the best way to track a future troubleshooting entry, then? I close this after that's tracked elsewhere.

Reply:
@vikram-redhat ^^^ ?